packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/services/migration/MigrationParticipantTests.java

```diff
@@ -82,18 +83,18 @@ public class MigrationParticipantTests extends AbstractIntegrationTests {
 
     @Test
     @DisplayName("Given a project with an old model, NodeDescriptionLabelExpressionMigrationParticipant migrates the model correctly")
-    @Sql(scripts = {"/scripts/migration.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
-    @Sql(scripts = {"/scripts/cleanup.sql"}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
+    @Sql(scripts = { "/scripts/migration.sql" }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
You will need your own test with a model already migrated with NodeDescriptionLabelExpressionMigrationParticipant to test only your migration

Reply:
I added my own test for NodeDescriptionLabelExpressionMigrationParticipant.
I also correct an bug, when there was several migration participant the ` .sorted(Collections.reverseOrder())` instruction crash, cause IMigrationParticipant is not `Comparable`